src/test/librados_test_stub/TestMemIoCtxImpl.h

```diff
@@ -78,8 +78,8 @@ class TestMemIoCtxImpl : public TestIoCtxImpl {
 private:
   TestMemIoCtxImpl(const TestMemIoCtxImpl&);
 
-  TestMemRadosClient *m_client;
-  TestMemCluster::Pool *m_pool;
+  TestMemRadosClient *m_client = NULL;
+  TestMemCluster::Pool *m_pool = NULL;
 
```

Comment:
I don't find an existing comparison with `NULL` in the implementation. So better change it to `nullptr` (introduce now).